package.json

```diff
@@ -45,5 +45,8 @@
     "parcel-bundler": "^1.12.4",
     "sourcegraph": "^24.7.0",
     "typescript": "^4.0.3"
+  },
+  "dependencies": {
+    "rxjs": "^6.6.3"
```

Comment:
I think this extension might get away without Rx (just async/await)

Reply:
Yeah, the only reason I imported it was so I could return multiple hovers. Could we bring `observableFromAsyncIterator` from codeintellify to Sourcegraph so users can return multiple hovers without bloating bundles?

Reply:
Yep, add it to https://docs.google.com/document/d/1l0grJVcTLdKGhRZD2LUMjB2e95xhnhBIE-oX5D_LAZQ/edit !